packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -147,6 +192,20 @@ class DestinationFromServiceRetriever {
     return withProxySetting;
   }
 
+  private static checkDestinationForCustomJwt(destination: Destination): void {
+    if (!destination.jwks && !destination.jwksUri) {
+      throw new Error(
+        'Failed to verify the JWT with no JKU! Destination must have `x_user_token.jwks` or `x_user_token.jwks_uri` property.'
+      );
+    }
+  }
+
+  private static isUserJwt(
+    token: SubscriberTokens | undefined
+  ): token is CustomToken | XsuaaToken {
+    return !!token && token.type !== 'iss';
```

Comment:
[req] You can shorten this.
```suggestion
    return token?.type !== 'iss';
```